volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -4,10 +4,13 @@
 
 import collections.abc
 import logging
+import stat
+from datetime import datetime
 import socket as socket_module
 from typing import Generator, Iterable, Iterator, Optional, Tuple, List
 
 from volatility3.framework import constants, exceptions, objects, interfaces, symbols
+from volatility3.framework import renderers
```

Comment:
This shouldn't be necessary (and it's because of trying to return a `BaseAbsentValue` descendant.  Please remove this when you get rid of the absent value return below.

Reply:
This is still needed for `renderers.conversion.unixtime_to_datetime()`

Reply:
I will import from `volatility3.framework.renderers import conversion` instead

Reply:
We probably ought to move that out of renderers, if it's a general purpose function?

Reply:
Maybe, `framework.objects.utility`?

Reply:
sure, but can do it in another PR? `wintime_to_datetime()` is used in many Windows related files

Reply:
Hehehe, yeah, no problem (although good if you could include a FIXME line just so we remember please)